demo/src/main/java/com/google/android/exoplayer2/demo/player/RtmpDataSource.java

```diff
@@ -0,0 +1,55 @@
+package com.google.android.exoplayer2.demo.player;
```

Comment:
You need to put the standard header present in other ExoPlayer source files at the top here